### ISSUE TITLE
[bug?] 'resizeToPreSnap' not working out-of-the-box when using snap callback

### ISSUE THREAD

**Issue by fq-selbach** [Flyer53/jsPanel4#80]
Just something I noticed and thought I should share it :-)

**Expected behavior:**
When using `resizeToPreSnap: true` with `snapLeftTop: function(...)` the window will resize when unsnapped.
**Observed behavior:**
It will not resize out-of-the-box but needs an additional line of code inside the callback.
**Fix:**
Include the line `panel.snapped = panel.snappableTo` in your callback, like this:
```
resizeToPreSnap:  true,
snapLeftTop: function(panel){
    panel.resize('50%').reposition('left-top');
    panel.snapped = panel.snappableTo;
}
```

**Comment by Flyer53** [comment]
Confirmed! Thanks for the report.
I think I know where the issue comes from and will probably have it fixed with the next update.

**Comment by Flyer53** [comment]
fixed with v4.6.0
